docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/NormalizeMinMaxMulticolumn.cs

```diff
@@ -0,0 +1,133 @@
+                    new InputOutputColumnPair("Features"),
+                    new InputOutputColumnPair("Features2")
+                },
+                fixZero: false);
+
+            // Normalize rows by finding min and max values in each row slot, but
+            // make sure zero values would remain zero after normalization. Helps
+            // preserve sparsity.
+            var normalizeFixZero = mlContext.Transforms.NormalizeMinMax(new[] {
+                    new InputOutputColumnPair("Features"),
+                    new InputOutputColumnPair("Features2")
+                },
+                fixZero: true);
+
+            // Now we can transform the data and look at the output to confirm the
+            // behavior of the estimator. This operation doesn't actually evaluate
+            // data until we read the data below.
+            var normalizeTransform = normalize.Fit(data);
+            var transformedData = normalizeTransform.Transform(data);
+            var normalizeFixZeroTransform = normalizeFixZero.Fit(data);
+            var fixZeroData = normalizeFixZeroTransform.Transform(data);
+            var column = transformedData.GetColumn<float[]>("Features").ToArray();
+            var column2 = transformedData.GetColumn<float[]>("Features2").ToArray();
+
+            for (int i = 0; i < column.Length; i++)
+                Console.WriteLine(string.Join(", ", column[i].Select(x => x
+                .ToString("f4"))) + "\t\t" + 
+                string.Join(", ", column2[i].Select(x => x.ToString("f4"))));
+
+            // Expected output:
+            // Features                                Features2  
+            // 0.6667, 0.6667, 1.0000, 0.0000          0.0000, 0.2857, 0.0000
+            // 1.0000, 1.0000, 0.7500, 0.0000          0.2500, 0.5714, 0.4000
+            // 0.3333, 0.3333, 0.5000, 0.0000          0.6250, 1.0000, 1.0000
+            // 0.0000, 0.0000, 0.0000, 1.0000          1.0000, 0.0000, 0.2000
+
+            var columnFixZero = fixZeroData.GetColumn<float[]>("Features").ToArray();
+            var column2FixZero = fixZeroData.GetColumn<float[]>("Features2").ToArray();
+
+            Console.WriteLine(Environment.NewLine);
+
+            for (int i = 0; i < column.Length; i++)
+                Console.WriteLine(string.Join(", ", columnFixZero[i].Select(x => x
+                .ToString("f4"))) + "\t\t" +
+                string.Join(", ", column2FixZero[i].Select(x => x.ToString("f4"))));
+
+            // Expected output:
+            // Features                                Features2  
+            // 0.5000, 0.5000, 1.0000, 0.0000          0.1111, 0.2857, 0.3750
+            // 1.0000, 1.0000, 0.6667, 0.0000          0.3333, 0.5714, 0.6250
+            // 0.0000, 0.0000, 0.3333, 0.0000          0.6667, 1.0000, 1.0000
+            // -0.5000, -0.5000, -0.3333, 1.0000       1.0000, 0.0000, 0.5000
+
+            // Let's get transformation parameters. Since we work with only one
```

Comment:
Since we work with only one -> Since there is only one

Reply:
Don't we have two columns here? Features and Features2?

Reply:
Updated the wording of the comment, but I'll need to see how the formula changes since there are two columns instead of one.